packages/flutter/lib/src/rendering/box.dart

```diff
@@ -15,7 +15,7 @@ import 'object.dart';
 
 // This class should only be used in debug builds.
 class _DebugSize extends Size {
```

Comment:
unrelated to changes in this PR: I wonder if the existence of this subclass causes the AOT compiler to believe that `Size` is polymorphic and makes it slower than it could be.

@rmacnak-google?